paths.py

```diff
@@ -24,3 +24,4 @@
 ROOT_DIR = os.path.dirname(SCRIPTS_DIR)
 TESTS_DIR = os.path.join(SCRIPTS_DIR, "tests")
 SERVICE_DIR = os.path.join(SCRIPTS_DIR, "service")
+PLUGINS_DIR = os.path.join(SCRIPTS_DIR, "plugins")
```

Comment:
Use this new constant when initialising the `InContext` instance in `incontext.py`.